integration_tests/mockApis/orchestration.ts

```diff
@@ -210,6 +212,7 @@ export default {
           prisonId: { equalTo: prisonId },
           prisonerId: { equalTo: prisonerId },
           visitors: { equalTo: visitorIds.join(',') },
+          username,
```

Comment:
```suggestion
          username: { equalTo: username },
```
I think this is probably why the tests are failing